sycl/doc/MultiTileCardWithLevelZero.md

```diff
@@ -143,7 +143,7 @@ try {
 ``` C++	
 try {
   // The queue is attached to the root-device, driver distributes to sub-devices, if any.
-  auto D = device(gpu_selector{});
+  auto D = device(gpu_selector_v);

```

Comment:
Do you really need to get the device?

Reply:
Good point! Selector argument has been moved to queue and this line has been removed.